README.md

```diff
@@ -11,7 +11,7 @@ Important security headers for Fastify. It is a tiny wrapper around
 
 ## Install
 ```
-npm i fastify-helmet
+npm install --save fastify-helmet
```

Comment:
```suggestion
npm i fastify-helmet
```

Isn't "--save" redundant as of npm v5?

Reply:
Oh right, If you didn't point that fact I have my habits with `-s` and `--save` when I install a package that I would have never realized that it now automatically save the installed package in the lock file :stuck_out_tongue: